docs/guides/component-testing/third-party-definitions.mdx

```diff
@@ -0,0 +1,245 @@
+  `,
+});
+```
+
+Our Framework Definition shows up in Cypress!
+
+<DocsImage
+  src="/img/guides/component-testing/framework-definition-1.png"
+  alt="custom framework definition"
+/>
+
+We defined the `dependencies`, which are also correctly handled - we haven't installed them all, so Cypress is prompting us to do so:
+
+
+<DocsImage
+  src="/img/guides/component-testing/framework-definition-2.png"
+  alt="required dependencies"
+/>
+
+## Mount Adapter
+
+The second part of defining a Framework Definition is the Mount Adapter. This is the function that renders the component in your tests using `cy.mount()`. 
+
+By default, Cypress will look for this as a `mount` function that is a **named export** from the package. This should be written in a `index.mjs` file. This example is for a [Solid.js](https://www.solidjs.com/) mount adapter:
+
+```js
+import { getContainerEl, setupHooks } from "@cypress/mount-utils";
+import { render } from "solid-js/web";
+
+let dispose;
+
+function cleanup() {
+  dispose?.();
+}
+
+/**
+ * @param {() => JSX.Element} - component to render
+ */
+export function mount(component, options = {}) {
+  const root = getContainerEl();
+  dispose = render(() => component, root);
+
+  return cy.wait(0, { log: false }).then(() => {
+    if (options.log) {
+      Cypress.log({
+        name: "mount",
+        message: "Mounted component",
+      });
+    }
+  });
+}
+
+// Cleanup after each test
+setupHooks(cleanup);
+```
+
+This is different for each library, but the concept is the same - identify how to mount or render a component in your library, and implement it in a function named `mount` in `index.mjs`.
+
+When a user configures Component Testing with your Framework Definition, their `supportFile` will contain the correct `mount` function:
+
+```js
+import { mount } from '@lmiller1990/cypress-ct-solid-js'
+
+Cypress.Commands.add('mount', mount)
+```
+
+## package.json
+
+The final thing you need is a correctly configured `package.json`. This example was created using [our official template](https://github.com/cypress-io/cypress-ct-definition-template).
+
+The main things to note is the `name` property, which follows the `@org/cypress-ct-*` convention, and is the `exports` property. The `node` entry points to the Framework Definition, and the `default` entry points to the Mount Adapter:
+
+```json {2,5-8}
+{
+  "name": "@lmiller1990/cypress-ct-solid-js",
+  "version": "0.0.4",
+  "description": "Example Framework Definition for Cypress and Solid.js",
+  "exports": {
+    "node": "./definition.cjs",
+    "default": "./index.mjs"
+  },
+  "files": [
+    "package.json",
+    "definition.cjs",
+    "index.mjs"
+  ],
+  "dependencies": {
+    "@cypress/mount-utils": "^4.0.0"
+  },
+  "devDependencies": {
+    "solid-js": "^1.6.0"
+  },
+  "peerDependencies": {
+    "solid-js": "^1.6.0"
+  }
+}
+```
+
```

Comment:
Worth adding a blurb around writing tests? Even if we don't have examples/content might be nice to at least suggest it as a good idea

Reply:
I'll add some tests to the example repo so people have an idea of where to start.

Reply:
Done https://github.com/cypress-io/cypress-documentation/pull/5069/commits/284dd59b64672a922a18a85fc6ab1a2b18aae138